proto/internal/temporal/server/api/persistence/v1/executions.proto

```diff
@@ -205,6 +205,7 @@ message VisibilityTaskInfo {
     google.protobuf.Timestamp visibility_time = 7 [(gogoproto.stdtime) = true];
     google.protobuf.Timestamp close_time = 8 [(gogoproto.stdtime) = true];
     google.protobuf.Timestamp start_time = 9 [(gogoproto.stdtime) = true];
+    int64 close_visibility_task_id = 10;
```

Comment:
I would rename this to a more general name in case in the future we want to ensure order between other task types.